grammar.js

```diff
@@ -1777,6 +1789,30 @@ module.exports = grammar({
       )
     },
 
+    global_call: $ => {
```

Comment:
I think it would be better to fit this into the current `call` rule:

```
    _global_method: $ => {
      const method = field('method', choice(
        $.identifier,
        alias($.identifier_method_call, $.identifier),
      ))

      return seq('::', method)
    },

    call: $ => {
      const receiver_call = choice(
        $._dot_call,
        field('method', alias($.identifier_method_call, $.identifier)),
        $._global_method,
      )
      // ...
    }